build.bash

```diff
@@ -2,7 +2,7 @@
 set -e
 
 ####################################################################
-#### dummy: changed this line 4 times to force another image build
+#### dummy: changed this line 5 times to force another image build
```

Comment:
😅

Reply:
image is intentionally only built when build.bash is changed

Reply:
If nothing else, we developers are certainly a lazy creative bunch. 

> Whenever there is a hard job to be done I assign it to a lazy man; he is sure to find an easy way of doing it. - Walter Chrysler

Reply:
Ok Walter, I got a job for you #1078

Reply:
I remember times when I would regularly remove and add the period at the end of a comment. :)